src/styles/_catalog.scss

```
@@ -1,22 +1,24 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  gap: 48px;
 
-  max-width: 944px;
-  margin: 0 auto;
+  grid-template-columns: repeat(1, $card-width);
```

no need to repeat one column
```suggestion
  grid-template-columns: $card-width;
```